hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -322,4 +262,113 @@ EthTxData replaceTo(byte[] to) {
                 r,
                 s);
     }
+
+    /**
+     * Encodes the transaction data into a EthTxData according to legacy RLP format.
+     *
+     * @return the encoded transaction data
+     */
+    private static EthTxData populateLegacyEthTxData(RLPItem rlpItem, byte[] rawTx) {
+        List<RLPItem> rlpList = rlpItem.asRLPList().elements();
+        if (rlpList.size() != 9) {
+            return null;
+        }
+
+        byte[] chainId = null;
+        byte[] v = rlpList.get(6).asBytes();
```

Comment:
Can we directly use the value rlpList.get(6).asBytes() below and not declare this variable?

Reply:
There is specific logic for `chainId` a few lines below, and I left it as it was.